diff --git a/build-tools/src/main/kotlin/org/jetbrains/kotlin/benchmark/CompileBenchmarkingPlugin.kt b/build-tools/src/main/kotlin/org/jetbrains/kotlin/benchmark/CompileBenchmarkingPlugin.kt
index 3df6e444b..4040cadff 100644
--- a/build-tools/src/main/kotlin/org/jetbrains/kotlin/benchmark/CompileBenchmarkingPlugin.kt
+++ b/build-tools/src/main/kotlin/org/jetbrains/kotlin/benchmark/CompileBenchmarkingPlugin.kt
@@ -70,7 +70,7 @@ open class CompileBenchmarkingPlugin : Plugin<Project> {
                         isIgnoreExitValue = true
                         konanRun.dependsOn(this)
                         doLast {
-                            exitCodes[name] = execResult.exitValue
+                            exitCodes[name] = execResult!!.exitValue
                         }
                     }
                 }
