diff --git a/libclangext/build.gradle b/libclangext/build.gradle
index 0dd79e3dd..dd5dd3c35 100644
--- a/libclangext/build.gradle
+++ b/libclangext/build.gradle
@@ -42,7 +42,7 @@ model {
             binaries.withType(StaticLibraryBinarySpec) { binary ->
                 if (!project.parent.convention.plugins.platformInfo.isWindows())
                     cppCompiler.args "-fPIC"
-                cppCompiler.args "--std=c++14", "-g", "-I${llvmDir}/include"
+                cppCompiler.args "--std=c++11", "-g", "-I${llvmDir}/include"
                 if (isEnabled) {
                     cppCompiler.args '-DLIBCLANGEXT_ENABLE=1'
                 }
